src/chocolatey.install/chocolatey.wxs

```diff
@@ -35,17 +31,13 @@
     <!-- Custom Actions to Install Chocolatey -->
     <SetProperty Id="InstallChocolatey"
       Before="InstallChocolatey"
-      Sequence="execute" 
+      Sequence="execute"
       Value="&quot;[SystemFolder]\WindowsPowerShell\v1.0\powershell.exe&quot; -NoProfile -ExecutionPolicy Bypass -Command &quot;$env:ChocolateyInstall='[INSTALLDIR]'; &amp; '[#ChocolateyInstall]' -ChocolateyDownloadUrl '[#ChocolateyNupkg]' -UseNativeUnzip&quot;" />
     <CustomAction Id="InstallChocolatey" BinaryKey="WixCA" DllEntry="WixQuietExec" Execute="deferred" Return="check" Impersonate="yes" />
 
     <!-- Custom Additions to the InstallExecuteSequence (e.g. the actions above) -->
     <InstallExecuteSequence>
       <Custom Action="InstallChocolatey" After="InstallFiles"><![CDATA[NOT Installed]]></Custom>
-      <!-- Set these values during the uninstall such that we can remove the folders -->
-      <Custom Action="SetHelpersDirectoryPath" Before="CostInitialize"><![CDATA[Installed]]></Custom>
-      <Custom Action="SetRedirectsDirectoryPath" Before="CostInitialize"><![CDATA[Installed]]></Custom>
-      <Custom Action="SetToolsDirectoryPath" Before="CostInitialize"><![CDATA[Installed]]></Custom>
```

Comment:
Same as above.

Reply:
These custom actions only fire when Chocolatey is already installed, which only happens on an upgrade, change, repair, or uninstall. None of which we currently do anything with.